src/containers/issue.py

```diff
@@ -0,0 +1,100 @@
+import re
+from typing import List, Optional
+
+from .repository import Repository
+from .milestone import Milestone
+
+
+class Issue:
+    def __init__(self, repository: Repository):
+        self.number: int = 0
+        self.organization_name: str = repository.organization_name
+        self.repository_name: str = repository.repository_name
+        self.title: str = ""
+        self.state: str = ""
+        self.url: str = ""
+        self.body: str = ""
+        self.created_at: str = ""
+        self.updated_at: str = ""
+        self.closed_at: Optional[str] = None
+        self.milestone_number: Optional[int] = None
+        self.milestone_title: Optional[str] = None
+        self.milestone_html_url: Optional[str] = None
+        self.labels: List[str] = []
+        self.page_filename: str = ""
+
+    def to_dict(self):
+        return {
+            "number": self.number,
+            "organization_name": self.organization_name,
+            "repository_name": self.repository_name,
+            "title": self.title,
+            "state": self.state,
+            "url": self.url,
+            "body": self.body,
+            "created_at": self.created_at,
+            "updated_at": self.updated_at,
+            "closed_at": self.closed_at,
+            "milestone_number": self.milestone_number,
+            "milestone_title": self.milestone_title,
+            "milestone_html_url": self.milestone_html_url,
+            "labels": [str(label) for label in self.labels],
+            "page_filename": self.page_filename
+        }
+
+    def load_from_json(self, issue):
```

Comment:
MIssing type of issue parameter.

Reply:
https://github.com/AbsaOSS/living-doc-generator/pull/16#discussion_r1680641697